src/features/movies/MovieTile/styled.js

```diff
@@ -0,0 +1,130 @@
+import styled from "styled-components";
+import { ReactComponent as star } from "./star.svg";
```

Comment:
That's an interesting solution ! Now I know what you meant in yesterday's call - I'll try it in the navigation 👍